build.sbt

```
@@ -95,16 +45,25 @@ lazy val `monster-ingester` = project
 
 lazy val `jadeclient` = project
   .in(file("jade_client"))
```

```suggestion
  .in(file("jade-client"))
```
Reasoning:
1. Depending on rendering, underscores can look like spaces in hyperlinks
2. Best to align with the sub-project name